lib/slide_rule/distance_calculators/day_of_month.rb

```diff
@@ -8,6 +8,7 @@ class DayOfMonth
       #   Does not take into account the number of days in the actual month being considered.
       #
       def calculate(first, second)
+        return if first.nil? || second.nil?
```

Comment:
Should we return nil explicitly here? I prefer to do it that way if I expect a nil from the method.